### PR TITLE
Fetch slots from VSP behind a flag

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This hooks up the slots call in order to fetch them from VSP.

Currently the date range fetch for slots doesn't work, but I tested this against a single date that returns four slots and it seems ok.

## Testing done
Local and unit testing

## Acceptance criteria
- [ ] Slots are fetched from VSP

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/containers/DateTimeSelectPage.jsx
Comment (jbalboni) [inline src/applications/vaos/containers/DateTimeSelectPage.jsx:35]: This is how I handled the slots coming from VSP in UTC and in facility time from VAR.

I'm hesitant to convert the facility times from VAR to UTC because I don't know if there is some weird timezone conversion logic in the backend that puts them in facility time in the first place. At least this way I'm not messing with the current slots and it's less likely to break for now.

File: src/applications/vaos/reducers/newAppointment.js
Comment (jbalboni) [inline src/applications/vaos/reducers/newAppointment.js:656]: The old call in some cases would return 5 digit codes, this should always be 3 digits.

**Review by narin — APPROVED** [review]
LGTM but we should re-validate once date ranges are working.

**Comment by jbalboni** [comment]
@narin Made a note in here that appt submission is blocked until that's fixed: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9066

**Pull request merged**
